ArcGISRuntimeSDKQt_QMLSamples/Analysis/FindFeaturesUtilityNetwork/FindFeaturesUtilityNetwork.qml

```diff
@@ -0,0 +1,358 @@
+// [WriteFile Name=FindFeaturesUtilityNetwork, Category=Analysis]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Layouts 1.12
+import QtQuick.Controls 2.12
+import Esri.ArcGISRuntime 100.7
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property url featureLayerUrl: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer"
+    property UtilityElement element
+    property UtilityNetworkSource utilityNetworkSource
```

Comment:
I'm surprised this doesn't throw an error since it is registered in our QML plugin as an uncreatable type.

Reply:
Same comment for a few of the below types (UtilityTerminal, ArcGISFeature, UtilityElementTraceResult). If it works, then that's fine. But I thought in the past, specifying type for uncreatable types would throw errors

Reply:
I think I will just change these to be on the safe side.